eng/wpf-debug.targets

```
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <!--
+    Copy PropertyGroup and Import item into the csproj file.
+    Update the path, as per the path to repo.
+
+    <PropertyGroup>
+      <WpfRepoRoot>C:\wpf</WpfRepoRoot>
+    </PropertyGroup>
+    <Import Project="$(WpfRepoRoot)\eng\wpf-debug.targets" />
+    -->
+
+  <PropertyGroup>
+    <WpfConfig>Debug</WpfConfig>
```

```suggestion
    <WpfConfig Condition="'$(WpfConfig)'==''">$(Configuration)</WpfConfig>
```

@ThomasGoulet73 How about the default is `$(Configuration)`

---

Picking up the default of consuming project is reasonable, when, SDK and Project, both build on configuration change. That is not true in this case.